google-home-community.groovy

```diff
@@ -2622,9 +2665,17 @@ private deviceStateForTrait_EnergyStorage(deviceTrait, device) {
 private deviceStateForTrait_FanSpeed(deviceTrait, device) {
     def currentSpeed = device.currentValue(deviceTrait.currentSpeedAttribute)
 
-    return [
-        currentFanSpeedSetting: currentSpeed
-    ]
+	if (deviceTrait.supportsFanSpeedPercent) {
+		def currentSpeedPercent = hubitatPercentageToGoogle(device.currentValue(deviceTrait.currentFanSpeedPercent))
+		return [
+			currentFanSpeedSetting: currentSpeed,
+			currentFanspeedPercent: currentSpeedPercent
```

Comment:
This could be the source of the slider issue.  It looks like the `currentFanSpeedPercent` attribute has slightly incorrect case.

```suggestion
			currentFanSpeedPercent: currentSpeedPercent
```